src/components/index.js

```diff
@@ -48,10 +48,10 @@ export {
   NetworkStatus,
   NodeInfo,
   NodeInfoBox,
+  TxHistory,
   Pulse,
   RpcTester,
   Spinner,
   TokenListForItem,
-  TxHistory,
```

Comment:
Not a strong preference, but these were alphabetized.

Reply:
Ah didn't know, will re-alphabetize